### PR TITLE
ADEN-9283 Migrate Prebid wgVariables to ICBM

### PR DESCRIPTION
**Pull request by Bielik20**

### CODE REVIEW

File: platforms/shared/bidders/setup-bidders.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { Context, InstantConfigService } from '@wikia/ad-engine';
```
Comment (fraszczakszymon) [inline platforms/shared/bidders/setup-bidders.ts]: Could you please add some comment that imports in this module should be used carefully? In order to avoid loading unexpected code in app/mobile-wiki

File: platforms/shared/bidders/setup-bidders.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+export function setupBidders(context: Context, instantConfig: InstantConfigService): void {
+	const hasFeaturedVideo = context.get('custom.hasFeaturedVideo');
+
+	if (hasFeaturedVideo) {
+		context.set('templates.stickyTLB.enabled', false);
```
Comment (fraszczakszymon) [inline platforms/shared/bidders/setup-bidders.ts]: This is not related to bidders at all

File: platforms/shared/bidders/setup-bidders.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+		context.set(
+			'bidders.prebid.rubicon_display.enabled',
+			instantConfig.get('icPrebidRubiconVideo'),
+		);
+
+		context.set('bidders.prebid.bidsRefreshing.enabled', context.get('options.slotRepeater'));
+		context.set('custom.rubiconInFV', instantConfig.get('icPrebidRubicon') && hasFeaturedVideo);
+
+		if (!instantConfig.get('icPrebidLkqdOutstream')) {
+			context.remove('bidders.prebid.lkqd.slots.INCONTENT_PLAYER');
```
Comment (fraszczakszymon) [inline platforms/shared/bidders/setup-bidders.ts]: Is this correct? Incontent player on mobile-wiki is lowercase. Also, I have mixed feelings about putting here hardcoded slot names.

File: platforms/shared/bidders/setup-bidders.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+		}
+
+		if (!instantConfig.get('icPrebidPubmaticOutstream')) {
+			context.remove('bidders.prebid.pubmatic.slots.INCONTENT_PLAYER');
```
Comment (fraszczakszymon) [inline platforms/shared/bidders/setup-bidders.ts]: Ditto

File: platforms/shared/bidders/setup-bidders.ts [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+		context.set('bidders.prebid.vmg.enabled', instantConfig.get('icPrebidVmg'));
+		context.set('bidders.prebid.appnexusAst.enabled', instantConfig.get('icPrebidAppNexusAst'));
+		context.set('bidders.prebid.rubicon.enabled', instantConfig.get('icPrebidRubicon'));
+		context.set(
+			'bidders.prebid.rubicon_display.enabled',
+			instantConfig.get('icPrebidRubiconVideo'),
+		);
+
+		context.set('bidders.prebid.bidsRefreshing.enabled', context.get('options.slotRepeater'));
+		context.set('custom.rubiconInFV', instantConfig.get('icPrebidRubicon') && hasFeaturedVideo);
```
Comment (fraszczakszymon) [inline platforms/shared/bidders/setup-bidders.ts]: Is this true? Shouldn't we use icPrebidRubiconVideo here?

File: platforms/shared/bidders/setup-bidders.ts [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+		context.set('bidders.prebid.lkqd.enabled', instantConfig.get('icPrebidLkqd'));
+		context.set('bidders.prebid.onemobile.enabled', instantConfig.get('icPrebidOneMobile'));
+		context.set('bidders.prebid.openx.enabled', instantConfig.get('icPrebidOpenX'));
+		context.set('bidders.prebid.pubmatic.enabled', instantConfig.get('icPrebidPubmatic'));
+		context.set('bidders.prebid.vmg.enabled', instantConfig.get('icPrebidVmg'));
+		context.set('bidders.prebid.appnexusAst.enabled', instantConfig.get('icPrebidAppNexusAst'));
+		context.set('bidders.prebid.rubicon.enabled', instantConfig.get('icPrebidRubicon'));
+		context.set(
+			'bidders.prebid.rubicon_display.enabled',
+			instantConfig.get('icPrebidRubiconVideo'),
```
Comment (fraszczakszymon) [inline platforms/shared/bidders/setup-bidders.ts]: icPrebidRubicon

File: platforms/shared/bidders/setup-bidders.ts [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+		context.set('bidders.prebid.gumgum.enabled', instantConfig.get('icPrebidGumGum'));
+		context.set('bidders.prebid.indexExchange.enabled', instantConfig.get('icPrebidIndexExchange'));
+		context.set('bidders.prebid.kargo.enabled', instantConfig.get('icPrebidKargo'));
+		context.set('bidders.prebid.lkqd.enabled', instantConfig.get('icPrebidLkqd'));
+		context.set('bidders.prebid.onemobile.enabled', instantConfig.get('icPrebidOneMobile'));
+		context.set('bidders.prebid.openx.enabled', instantConfig.get('icPrebidOpenX'));
+		context.set('bidders.prebid.pubmatic.enabled', instantConfig.get('icPrebidPubmatic'));
+		context.set('bidders.prebid.vmg.enabled', instantConfig.get('icPrebidVmg'));
+		context.set('bidders.prebid.appnexusAst.enabled', instantConfig.get('icPrebidAppNexusAst'));
+		context.set('bidders.prebid.rubicon.enabled', instantConfig.get('icPrebidRubicon'));
```
Comment (fraszczakszymon) [inline platforms/shared/bidders/setup-bidders.ts]: icPrebidRubiconVideo

File: platforms/shared/bidders/setup-bidders.ts (hunk 2 above)
Comment (Bielik20) [reply platforms/shared/bidders/setup-bidders.ts]: Yeah but `hasFeaturedVideo` was used here. I can change it though.

File: platforms/shared/bidders/setup-bidders.ts (hunk 6 above)
Comment (Bielik20) [reply platforms/shared/bidders/setup-bidders.ts]: rubicon is above this is video.

Comment (fraszczakszymon) [reply platforms/shared/bidders/setup-bidders.ts]: Discussed on slack. It should be icRubiconDisplay

File: platforms/shared/bidders/setup-bidders.ts (hunk 2 above)
Comment (fraszczakszymon) [reply platforms/shared/bidders/setup-bidders.ts]: Please do, I think it should stay in mobile-wiki/app repo

File: platforms/shared/bidders/setup-bidders.ts (hunk 6 above)
Comment (Bielik20) [reply platforms/shared/bidders/setup-bidders.ts]: icPrebidRubiconDisplay

File: platforms/shared/bidders/setup-bidders.ts (hunk 3 above)
Comment (Bielik20) [reply platforms/shared/bidders/setup-bidders.ts]: Yeah, great catch. I will move it to mobile-wiki/app for now but I would love to see it unified somehow.
